integration-testing/test/test_star_network.py

```diff
@@ -26,6 +26,7 @@ def star_network(system):
 def test_convergence(star_network):
     logging.info("Star network converged successfully.")
 
-@pytest.mark.skip(reason="This doesn't work since the show-blocks functionality was removed")
-def test_casper_propose_and_deploy(star_network):
-    rnode_testing.casper_propose_and_deploy.run(star_network)
+# TODO This test can't pass now because newly proposed blocks can not sync up in other nodes.
```

Comment:
If this enables a failing tests, it will fail in CI after merging to dev.

Reply:
I don't know why the node can not receive blocks. @KellyatPyrofex said it happened in the node test session,right? Maybe we should wait for some valid patch to fix it and then merge it. 

We can put a `don't merge` label here.